ansible/roles/local_storage/templates/cluster-monitoring-config.yml.j2

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+data:
+  config.yaml: |
+    prometheusOperator:
+      baseImage: quay.io/coreos/prometheus-operator
+      prometheusConfigReloaderBaseImage: quay.io/coreos/prometheus-config-reloader
+      configReloaderBaseImage: quay.io/coreos/configmap-reload
+    prometheusK8s:
```

Comment:
Do we not need a nodeselector / taint/toleration to ensure that the Prometheus pod is scheduled on the node with local storage??

Reply:
I haven't seen this to be necessary on the shiftstack clusters I have been building but I went ahead and added the option to add a node selector via `openshift_prometheus_node_selector`.

Reply:
All of the other pods in `openshift-monitoring` also can take node selectors so we might want to just refactor any prom config into a separate playbook/role so each use case can configure what it needs without running through the local-storage playbook here too.